files/collaborative_document.md

```diff
@@ -0,0 +1,72 @@
+![](https://i.imgur.com/iywjz8s.png)
+
+
+# Collaborative Document
+
+YYYY-MM-DD Workshop Type.
+
+Welcome to The Workshop Collaborative Document.
+
+This Document is synchronized as you type, so that everyone viewing this page sees the same text. This allows you to collaborate seamlessly on documents.
+
+----------------------------------------------------------------------------
+
+This is the Document for today: [link](<url>)
+
+Collaborative Document day 1: [link](<url>)
+
+Collaborative Document day 2: [link](<url>)
+
+Collaborative Document day 3: [link](<url>)
+
+Collaborative Document day 4: [link](<url>) 
+
+## 👮Code of Conduct
+
+* Participants are expected to follow those guidelines:
+* Use welcoming and inclusive language.
+* Be respectful of different viewpoints and experiences.
+* Gracefully accept constructive criticism.
+* Focus on what is best for the community.
+* Show courtesy and respect towards other community members.
+ 
+## ⚖️ License
+
+All content is publicly available under the Creative Commons Attribution License: [creativecommons.org/licenses/by/4.0/](https://creativecommons.org/licenses/by/4.0/).
+
+## 🙋Getting help
+
+To ask a question, type `/hand` in the chat window.
+
+To get help, type `/help` in the chat window.
+
+You can ask questions in the document or chat window and helpers will try to help you.
+
+## 🖥 Workshop website
+
+[link](<url>)
+
+🛠 Setup
+
+[link](<url>)
+
+Download files
+
+[link](<url>)
+
+## 👩‍🏫👩‍💻🎓 Instructors
+
+Instructor 1, Instructor 2
+
+## 🧑‍🙋 Helpers
+
+Helper 1, Helper 2  
+
+## 👩‍💻👩‍💼👨‍🔬🧑‍🔬🧑‍🚀🧙‍♂️🔧 Roll Call
+Name/ pronouns (optional) / job, role / social media (twitter, github, ...) / background or interests (optional) / city
+
+## 🗓️ Agenda
+
+## 🧠 Collaborative Notes
+
```

Comment:
I would add here:
`
## 🔧 Exercises
`
(maybe you have ideas for a better icon than this tool)

Reply:
Sounds good, I can add it.